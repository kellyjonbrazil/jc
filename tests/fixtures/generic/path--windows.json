{
  "path": "C:\\Windows\\Program Files\\xfolder\\file.txt",
  "parent": "C:\\Windows\\Program Files\\xfolder",
  "filename": "file.txt",
  "stem": "file",
  "extension": "txt",
  "path_list": [
    "C:\\",
    "Windows",
    "Program Files",
    "xfolder",
    "file.txt"
  ]
}