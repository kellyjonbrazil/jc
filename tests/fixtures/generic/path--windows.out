C:\Windows\Program Files\xfolder\file.txt